src/sagas/walletConnect.js

```diff
@@ -316,6 +318,13 @@ export function* onSessionRequest(action) {
   } catch (e) {
     let shouldAnswer = true;
     switch (e.constructor) {
+      case BluePrintNotFoundError:
```

Comment:
note: I believe the name for this error will change to `BlueprintNotFoundError`, pointed in this [comment](https://github.com/HathorNetwork/hathor-rpc-lib/pull/16#discussion_r1725997886).